Podfile.lock

```diff
@@ -1,3 +1,16 @@
-PODFILE CHECKSUM: 7e3d3d38e3c6c64c581bdfbe83e25e0034c3de26
+PODS:
```

Comment:
I'm far from an expert, but why does `Prodfile.lock` need a reference to v17 of its own project?

Reply:
For the future, it would be good to have separate targets for the app (example) and library (seatsio) and use the library in the app for testing.

I'm using the development pod here to see how the pod will be generated as we use the same target for the library and example app.
https://guides.cocoapods.org/making/development-cocoapods.html 

I'll set up multiple targets as soon as I have a bit of free time!

Reply:
👍

Reply:
@mroloux , I've separated the example app and framework. Could you please review it once more? Also, could you set up the actions to skip running the pod install?

Reply:
Maybe it's better if you remove `pod.install` in this branch? Then we see if the build succeeds.

Reply:
Done